support-frontend/assets/helpers/paymentProviders.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { loadStripe, openDialogBox, setupStripeCheckout } from 'helpers/paymentIntegrations/stripeCheckout';
+import { type IsoCountry } from 'helpers/internationalisation/country';
+import { type PaymentAuthorisation } from 'helpers/paymentIntegrations/readerRevenueApis';
+
+import type {IsoCurrency} from "./internationalisation/currency";
+
+function showStripe(
```

Comment:
🥇 for moving these into their own module

Reply:
do the parameters look sensible? there were a lot of paper specific "super objects" being passed around, and I ended up doing quite a bit of unpacking hence the parameter list looks a bit long.  Also I ended up going around the houses learning how the architecture fits together, so I have ended up simplified but increased the quantity of the params in the product specific versions of `paymentProviders.js`.
They could probably be refactored back to the paper/weekly specific objects if that's better.

Reply:
Yes I think they're fine.